stubs/lang/en/resourceCollection.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
This file (and the corresponding French one) should be named in kebab case, e.g. `resource-collection`).